From 4ca6f9dc359dd27178675185a0b6d30d3345111f Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Fri, 22 Apr 2016 18:22:25 +0200
Subject: [PATCH 6/8] config: support shortcuts for vbox inside windows

---
 config.def.h | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/config.def.h b/config.def.h
index b0fe3f5..ff07891 100644
--- a/config.def.h
+++ b/config.def.h
@@ -73,12 +73,21 @@ static Key keys[] = {
 	{ MODKEY,                       XK_p,      spawn,          {.v = dmenucmd } },
 	{ MODKEY|ShiftMask,             XK_Return, spawn,          {.v = termcmd } },
 	{ MODKEY,                       XK_b,      togglebar,      {0} },
+
 	{ MODKEY,                       XK_j,      focusstack,     {.i = +1 } },
 	{ MODKEY,                       XK_k,      focusstack,     {.i = -1 } },
 	{ MODKEY,                       XK_i,      incnmaster,     {.i = +1 } },
 	{ MODKEY,                       XK_d,      incnmaster,     {.i = -1 } },
 	{ MODKEY,                       XK_h,      setmfact,       {.f = -0.05} },
 	{ MODKEY,                       XK_l,      setmfact,       {.f = +0.05} },
+        // support for virtualbox inside Windows
+	{ MODKEY,                       XK_Up,     focusstack,     {.i = +1 } },
+	{ MODKEY,                       XK_Down,   focusstack,     {.i = -1 } },
+	{ MODKEY,                       XK_KP_Add,      incnmaster,{.i = +1 } },
+	{ MODKEY,                       XK_KP_Subtract, incnmaster,{.i = -1 } },
+	{ MODKEY,                       XK_Left,   setmfact,       {.f = -0.05} },
+	{ MODKEY,                       XK_Right,  setmfact,       {.f = +0.05} },
+
 	{ MODKEY,                       XK_Return, zoom,           {0} },
 	{ MODKEY,                       XK_Tab,    view,           {0} },
 	{ MODKEY|ShiftMask,             XK_c,      killclient,     {0} },
-- 
2.12.0

