From 718a6e3ac757d147147fafe98bd04c1b74d23e9f Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Fri, 10 Mar 2017 09:35:34 +0100
Subject: [PATCH 9/9] qtpass as float

---
 config.def.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config.def.h b/config.def.h
index 1d9097e..a42da60 100644
--- a/config.def.h
+++ b/config.def.h
@@ -31,6 +31,7 @@ static const Rule rules[] = {
 	 */
 	{ "Gimp",     NULL,       NULL,       0,            1,           -1 },
 	{ "Xpra",     NULL,       NULL,       0,            1,           -1 },
+	{ "QtPass",     NULL,       NULL,       0,            1,           -1 },
 	{ "Firefox",  NULL,       NULL,       1 << 8,       0,           -1 },
 };
 
-- 
2.12.0

