From a4cdab467df1a2c16dc5a3c07126f2a03521b7be Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Mon, 12 Oct 2020 11:57:14 +0200
Subject: [PATCH 1/2] compile with new PrimOp struct

---
 extra-builtins.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/extra-builtins.cc b/extra-builtins.cc
index 862fe9b..1af288e 100644
--- a/extra-builtins.cc
+++ b/extra-builtins.cc
@@ -47,12 +47,12 @@ static void extraBuiltins(EvalState & state, const Pos & _pos,
             auto sExec = state.symbols.create("exec");
             auto vExec = state.allocAttr(*arg, sExec);
             vExec->type = tPrimOp;
-            vExec->primOp = new PrimOp(prim_exec, 1, sExec);
+            vExec->primOp = new PrimOp { .fun = prim_exec, .arity = 1, .name = sExec};
 
             auto sImportNative = state.symbols.create("importNative");
             auto vImportNative = state.allocAttr(*arg, sImportNative);
             vImportNative->type = tPrimOp;
-            vImportNative->primOp = new PrimOp(prim_importNative, 2, sImportNative);
+            vImportNative->primOp = new PrimOp { .fun = prim_importNative, .arity = 2, .name = sImportNative };
 
             arg->attrs->sort();
         }
-- 
2.28.0

