From 855cef4b3d112d5fa155b0d78287101c72cd2593 Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Thu, 22 Mar 2018 14:01:28 +0100
Subject: [PATCH 1/4] apply st-no_bold_colors-20170623-b331da5.diff

---
 x.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/x.c b/x.c
index c343ba2..42e7e9f 100644
--- a/x.c
+++ b/x.c
@@ -1295,7 +1295,7 @@ xdrawglyphfontspecs(const XftGlyphFontSpec *specs, Glyph base, int len, int x, i
 
 	/* Change basic system colors [0-7] to bright system colors [8-15] */
 	if ((base.mode & ATTR_BOLD_FAINT) == ATTR_BOLD && BETWEEN(base.fg, 0, 7))
-		fg = &dc.col[base.fg + 8];
+		fg = &dc.col[base.fg];
 
 	if (IS_SET(MODE_REVERSE)) {
 		if (fg == &dc.col[defaultfg]) {
-- 
2.16.2

