From 92bba189dbc2b1008467fe15b53c9f7d4693643d Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Tue, 3 May 2022 20:42:13 +0200
Subject: [PATCH 2/2] bigger occupied rectangle

---
 src/bar.cpp | 18 +++++++++++-------
 1 file changed, 11 insertions(+), 7 deletions(-)

diff --git a/src/bar.cpp b/src/bar.cpp
index fab5a8f..c9d6c30 100644
--- a/src/bar.cpp
+++ b/src/bar.cpp
@@ -245,13 +245,17 @@ void Bar::renderTags()
 			tag.state & TagState::Active ? colorActive : colorInactive,
 			tag.state & TagState::Urgent);
 		renderComponent(tag.component);
-		auto indicators = std::min(tag.numClients, static_cast<int>(_bufs->height/2));
-		for (auto ind = 0; ind < indicators; ind++) {
-			auto w = ind == tag.focusedClient ? 7 : 1;
-			cairo_move_to(_painter, tag.component.x, ind*2+0.5);
-			cairo_rel_line_to(_painter, w, 0);
-			cairo_close_path(_painter);
-			cairo_set_line_width(_painter, 1);
+		auto occ = tag.numClients > 0;
+		if (tag.numClients > 0) { // draw rectangle if occupied
+			auto boxs = barfont.height / 9;
+			auto boxw = barfont.height / 6 + 2;
+			cairo_move_to(_painter, tag.component.x, 0.5);
+			cairo_rel_line_to(_painter, boxw, 0);
+			cairo_rel_line_to(_painter, 0, boxs);
+			cairo_rel_line_to(_painter, -boxw, 0);
+			cairo_rel_line_to(_painter, 0, -boxs);
+			//cairo_close_path(_painter);
+			cairo_set_line_width(_painter, 2);
 			cairo_stroke(_painter);
 		}
 	}
-- 
2.35.3

