From 9ea255d4f81c28f3e7d393dd39f23f3c491cdbbd Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Sun, 3 Jan 2021 17:03:10 +0100
Subject: [PATCH 3/3] value: mkPrimOp

---
 extra-builtins.cc | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/extra-builtins.cc b/extra-builtins.cc
index 01f4a2f..ed0010e 100644
--- a/extra-builtins.cc
+++ b/extra-builtins.cc
@@ -46,13 +46,11 @@ static void extraBuiltins(EvalState & state, const Pos & _pos,
 
             auto sExec = state.symbols.create("exec");
             auto vExec = state.allocAttr(*arg, sExec);
-            vExec->type = tPrimOp;
-            vExec->primOp = new PrimOp { .fun = prim_exec, .arity = 1, .name = sExec};
+            vExec->mkPrimOp(new PrimOp { .fun = prim_exec, .arity = 1, .name = sExec});
 
             auto sImportNative = state.symbols.create("importNative");
             auto vImportNative = state.allocAttr(*arg, sImportNative);
-            vImportNative->type = tPrimOp;
-            vImportNative->primOp = new PrimOp { .fun = prim_importNative, .arity = 2, .name = sImportNative };
+            vImportNative->mkPrimOp(new PrimOp { .fun = prim_importNative, .arity = 2, .name = sImportNative });
 
             arg->attrs->sort();
         }
-- 
2.29.2

