From 57dccf0615397179332abcdf343b2f9736951624 Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Sun, 3 Jan 2021 17:05:47 +0100
Subject: [PATCH 4/4] mkStringNoCopy -> mkString

---
 extra-builtins.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/extra-builtins.cc b/extra-builtins.cc
index ed0010e..35f7e48 100644
--- a/extra-builtins.cc
+++ b/extra-builtins.cc
@@ -69,9 +69,9 @@ static void cflags(EvalState & state, const Pos & _pos,
     Value ** _args, Value & v)
 {
     state.mkAttrs(v, 3);
-    mkStringNoCopy(*state.allocAttr(v, state.symbols.create("NIX_INCLUDE_DIRS")), NIX_INCLUDE_DIRS);
-    mkStringNoCopy(*state.allocAttr(v, state.symbols.create("NIX_CFLAGS_OTHER")), NIX_CFLAGS_OTHER);
-    mkStringNoCopy(*state.allocAttr(v, state.symbols.create("BOOST_INCLUDE_DIR")), BOOST_INCLUDE_DIR);
+    mkString(*state.allocAttr(v, state.symbols.create("NIX_INCLUDE_DIRS")), NIX_INCLUDE_DIRS);
+    mkString(*state.allocAttr(v, state.symbols.create("NIX_CFLAGS_OTHER")), NIX_CFLAGS_OTHER);
+    mkString(*state.allocAttr(v, state.symbols.create("BOOST_INCLUDE_DIR")), BOOST_INCLUDE_DIR);
 }
 
 static RegisterPrimOp rp2("__nix-cflags", 0,
-- 
2.29.2

