From 9132fc3503fb426459e6efe5e1e41b67553b7c88 Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Wed, 22 Jun 2016 10:47:44 +0200
Subject: [PATCH 3/3] custom changes

---
 config.def.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config.def.h b/config.def.h
index 386375e..5de940c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Liberation Mono:pixelsize=14:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -62,8 +62,8 @@ static unsigned int cursorthickness = 2;
  */
 static int bellvolume = 0;
 
-/* default TERM value */
-static char termname[] = "st-256color";
+/* TERM value */
+static char termname[] = "xterm-256color";
 
 /*
  * spaces per tab
-- 
2.9.3

