From 9cbb5898bb8228f30ba4039fa1f550a4f581b6dc Mon Sep 17 00:00:00 2001
From: David Guibert <david.guibert@gmail.com>
Date: Thu, 22 Mar 2018 14:02:39 +0100
Subject: [PATCH 3/4] custom changes

---
 config.def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.def.h b/config.def.h
index c7ad56b..5e83342 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Droid Sans Mono:pixelsize=14:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -63,7 +63,7 @@ static unsigned int cursorthickness = 2;
 static int bellvolume = 0;
 
 /* default TERM value */
-char *termname = "st-256color";
+char *termname = "xterm-256color";
 
 /*
  * spaces per tab
-- 
2.16.2

